general-information.md

```diff
@@ -0,0 +1,48 @@
+# Mobility Data Specification: **General information**
+
+This document contains specifications that are shared between the various MDS APIs such as `agency`, `provider` and `policy`.
+
+## Table of Contents
+
+* [Versioning](#versioning)
+
+## Versioning
+
+MDS APIs must handle requests for specific versions of the specification from clients.
+
+Versioning must be implemented through the use of a custom media-type, `application/vnd.mds.provider+json`, combined with a required `version` parameter.
```

Comment:
We could remove the `.provider` part of the header since it is now available for all the APIs but it would make the changes breaking for provider.

This could be done later in the 0.5 timeframe if we want.

Reply:
Seems like a good idea, want to open an issue for tracking purposes?

Reply:
I have done it with #455